### PR TITLE
fix(parser): Don't crash eval'ing require in transpiled bundle

### LINKED ISSUE #9: Using with webpack
I'd like to use this in a webpacked project, i.e. create-react-app

Trying to import it:

```
import TOML from '@iarna/toml'
```

results in this error:

```
Uncaught ReferenceError: require is not defined
```

It's coming from here:

image

Do you have a suggestion for using this with webpack?

### PR DESCRIPTION
**Pull request by agriffis**
Fixes #9

### CODE REVIEW

File: lib/toml-parser.js [hunk 1]
```diff
@@ -162,9 +162,15 @@ function isList (obj) {
...
 // in an eval, to let bundlers not slurp in a util proxy
-const utilInspect = eval(`require('util').inspect`)
-/* istanbul ignore next */
-const _inspect = (utilInspect && utilInspect.custom) || 'inspect'
+let _custom
+try {
+  const utilInspect = eval("require('util').inspect")
+  _custom = utilInspect.custom
+}
+catch (e) {
```
Comment (iarna) [inline lib/toml-parser.js]: The linter insists that this be on the same line as the close `}`. (I'll fix this myself when merging, if you don't get to it first.)

**Review by iarna — APPROVED** [review]
Thank you, in retrospect this obviously is what I wanted.

**Comment by agriffis** [comment]
@iarna I pushed an update that moves the `catch` with the close `}`

**Comment by agriffis** [comment]
@iarna Is there anything more you need from me on this?

**Comment by iarna** [comment]
The tests do need to pass before I merge—but don't worry about it, I'll take care of that now and merge.

File: lib/toml-parser.js [hunk 2]
```diff
@@ -162,9 +162,14 @@ function isList (obj) {
 }
 
 // in an eval, to let bundlers not slurp in a util proxy
-const utilInspect = eval(`require('util').inspect`)
-/* istanbul ignore next */
```
Comment (iarna) [inline lib/toml-parser.js:166]: Deleting the instanbul directive results in tests failing due to 172 not having coverage for the alternative.

**Pull request closed without merging**